app/src/main/java/dto/Pokemon.kt

```diff
@@ -1,5 +1,5 @@
 package dto
 
-data class Pokemon(var pokemonId: Int = 0, var name: String="", var description: String="", var types: ArrayList<String>) {
+data class Pokemon(var pokemonID: Int = 0, var name: String = "", var description: String = "", var types: ArrayList<String>) {
```

Comment:
I can go either way on this.  Some will capitalize an acronym, others will CamelCase it.